bin/build_tasks.sh

```diff
@@ -18,7 +18,7 @@ fi
 cd ../../
 
 failure=0
-for TASK in $(ls -d tasks/* | egrep -v "package|dr_dbutils|pg_utils|shared_libraries")
+for TASK in $(ls -d tasks/* | egrep -v "package|shared_libraries")
```

Comment:
`shared_libraries` should also be removed as it no longer resides in the `tasks` directory nor is it built there like the `packages` directory.

Reply:
Removed